idea-plugin/src/test/kotlin/arrow/meta/ide/plugins/proofs/annotators/CoercionAnnotatorTest.kt

```diff
@@ -31,7 +32,11 @@ class CoercionAnnotatorTest : IdeTestSetUp(
             collectAnnotations(code, myFixture)
           },
           result = resolvesWhen("CoercionAnnotatorTest1 for 1 implicit coercion property") { result: List<HighlightInfo> ->
-            result.size == 1
+            result.size == 1 &&
```

Comment:
this can be nicely expressed with resolveWith and `first()` followed with `takeIf`

Reply:
But there are also other options scope variables and using functions for common assertions